timemachine/fe/rbfe.py

```diff
@@ -775,6 +688,16 @@ def get_mean_final_barostat_volume_scale_factor(trajectories_by_state: Iterable[
         plots = make_pair_bar_plots(pair_bar_result, temperature, combined_prefix)
 
         stored_trajectories = []
+
+        if keep_idxs is None:
+            keep_idxs = [0, -1]  # keep frames from first and last windows
+        elif keep_idxs is np.inf:
```

Comment:
@mcwitt I had to modify this logic a bit to support another indicator value for `keep_idxs` - since we don't know the # of actual initial states (as it's determined by HREX), we instead need a way of saying "keep everything". `None` is already reserved for first and last windows, so I picked `np.inf` instead. Is this too hacky?

Reply:
Agreed that a special value for "all" is needed. How about just the string `"all"`? Can indicate this in the type as `Optional[List[int] | Literal["all"]]`.

Alternatively, maybe it makes more sense to reserve `None` for all states, and change the default value to `(0, -1)`?

Reply:
Okay I can try that - I'm hesitant about changing the default since it's API breaking. I'm not sure if upstream code depends on this in production already.

Reply:
To be clear, I'm not suggesting changing the default behavior if `keep_idxs` is not specified -- that would still be to return frames for the first and last states. It would just be to change the meaning of `None` (which is still potentially API-breaking).

Reply:
ugh - just realized we use `keep_idxs` for all the other `run_*` functions. Not sure if we want to make this change consistently or not.

Reply:
(changed to "all" per suggestion)